web/sites/default/settings.prod-eks.php

```diff
@@ -35,8 +34,10 @@
 $config['config_split.config_split.crm_prod']['status'] = TRUE;
 $config['config_split.config_split.crm_stage']['status'] = FALSE;
 
+if (getenv('CONNECT_DOMAIN')) {
```

Comment:
I think we probably want to set this before going live?

Reply:
Yep, on PaaS that looks like

```
CONNECT_DOMAIN: connect.nationalleadership.gov.uk
```